scripts/pr.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+
+import sys, os, json, subprocess, pip, getpass
+import __builtin__
+
+from github import Github
+from os.path import expanduser
+
+
+FileNotFoundError = getattr(__builtin__,"IOError","FileNotFoundError")
+
+home = expanduser("~")
+configPath = home + '/.pr_script_config.json'
+scriptPath = os.path.realpath(__file__)
+
+
+def requirePass():
+    return getpass.getpass("Type your Github password: ")
+
+
+def requireBase():
+    return raw_input("Please specify the base branch you want the changes pulled into: ")
+
+
+def requireTickedId():
+    return raw_input("Type JIRA ticked's id: ")
+
+
+def requireTickedHeader():
+    return raw_input("Type JIRA ticked's description: \n")
+
+
+def requireBody():
+    return raw_input("Type PR's body: \n")
+
+
+def requireRemote():
+    remote = raw_input("Please, specify the repository for PR's in format ':owner/:repo': \n")
+    return remote.lower()
+
+
+def requireFeatureName():
+    return raw_input("Type name of the feature you will work on: ").replace(" ", "-")
+
+
+def mapDict(f,d):
+    return dict(map(f, d.iteritems()))
+
+
+def validateValues(items):
+    (k,v) = items
+    if v is None:
+        print("Cannot infer the {0} value".format(k))
+        newV = raw_input("Please specify a value for the : {0}".format(k))
+        return (k,newV)
+    else:  
+        return (k,v)
+
+    
+def readFile(name):
+    f = open(name, 'a+')
+    content = f.read()
+    f.close()
+    return content
```

Comment:
Opening for read is probably sufficient:
```
with open(name, 'r') as f:
    return f.read()
```
which also lets the context manager handle the file resource.